src/pretix/control/templates/pretixcontrol/base.html

```diff
@@ -59,6 +59,7 @@
             <script type="text/javascript" src="{% static "fileupload/jquery.fileupload.js" %}"></script>
             <script type="text/javascript" src="{% static "lightbox/js/lightbox.min.js" %}"></script>
             <script type="text/javascript" src="{% static "are-you-sure/jquery.are-you-sure.js" %}"></script>
+            <script type="text/javascript" src="{% static "pretixcontrol/js/ui/popover.js" %}"></script>
```

Comment:
There is a potential issue here: Later on, if you modify *popover.js* and deploy, end user may still get the old content of the code, due to browser cache.
To force browser to load the latest version of the file, you need to make the file URL vary after each deployment. You can use this lib https://pypi.org/project/dj-static-revision/ to do that.

Reply:
Hi @hongquan, thanks a lot for your suggestion, as I checked, currently eventyay-tickets system using 'django.contrib.staticfiles.storage.ManifestStaticFilesStorage', The ManifestStaticFilesStorage class in Django already handles the issue of browser caching by appending a hash to the filenames of static files.
And our deployment steps included collectstatic, so the caching issue won't happen.